app/server/models/user.py

```diff
@@ -499,9 +502,16 @@ def is_resetting(self):
         is_resetting = len(self.password_reset_tokens) > 0
         return is_resetting
 
-    # TODO(ussd): change to a field once we figure out what's the deal with resetting
     def pin_failed_attempts(self):
```

Comment:
we can just remove this method and change to the field in places that use it

Reply:
Should be a field. I think there's some redundancy I need to remove here.